lib/secure_headers/configuration.rb

```diff
@@ -121,7 +119,8 @@ def deep_copy_if_hash(value)
     def initialize(&block)
       self.hpkp = OPT_OUT
       self.referrer_policy = OPT_OUT
-      self.csp = self.class.send(:deep_copy, CSP::DEFAULT_CONFIG)
+      self.csp = ContentSecurityPolicyConfig::DEFAULT
```

Comment:
Would smarter defaults be to swap these conditions and allow a report only by default?

```
self.csp = OPT_OUT
self.csp_report_only = ContentSecurityPolicyConfig::DEFAULT
```

Or would this break backwards compatibility with existing implementations?